CHANGELOG.rst

```diff
@@ -5,6 +5,21 @@ amazon.aws Release Notes
 .. contents:: Topics
 
 
+v5.5.2
+======
+
+Release Summary
+---------------
+
+This release brings few bugfixes.
+
+Bugfixes
+--------
+
+- Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
```

Comment:
Just to give a better understanding about the module that is affected by the change, can you please add it at the beggining?
```suggestion
- cloudwatchevent_rule - Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
```